rovilay/ideazz-coalition
# signup and login users

#### What does this PR do?
This PR covers the tasks outlined below 👇🏽

#### Description of Task to be completed?
- create users controllers
- create users routes

#### How should this be manually tested?
- git clone this app - `git clone https://github.com/rovilay/ideazz-coalition.git`
- navigate to the cloned app
- git checkout to this branch - `git checkout -b ft-users-auth`
- set up the `.env` with the `.env.sample` file
- install dependencies - `npm install`
- start the app - `npm run server`
- navigate to `localhost:port` on the browser
- you should see a welcome message - `Welcome to Welcome to Ideazz-coalitionAPI`
- signup a user using the `localhost:port/api/v1/users/signup` route. Provide `name, email and password`

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
